gprofiler/profilers/python.py

```diff
@@ -151,6 +153,49 @@ def __init__(
         self.output_path = Path(self._storage_dir) / f"pyperf.{random_prefix()}.col"
         self.user_stacks_pages = user_stacks_pages
 
+    @staticmethod
+    def _fix_kernel_headers_symlink():
+        try:
+            if get_run_mode() not in ("container", "k8s"):
+                return
+
+            try:
+                uname = subprocess.check_output(["uname", "-r"]).decode('latin-1').strip()
+            except subprocess.CalledProcessError:
+                return
+
+            build_path = f"/lib/modules/{uname}/build"
+            if not os.path.islink(build_path):
+                return
+
+            if os.path.lexists(build_path + "/"):
+                return
+
+            lib_modules_uname_path = f"/lib/modules/{uname}"
+
+            os.makedirs('/dev/shm/modules_mount/upper', 555)
```

Comment:
`/dev/shm` is used as a tmpfs here?

Reply:
Yes, for technical reasons, it didn't work on `/tmp` because that one isn't

Reply:
Yeah it should be on a tmpfs. `/tmp` is not a tmpfs on all distros and I suppose `/dev/shm` is. Could you drop a comment on that? it's not obvious, took me a moment to figure out why you do that

Reply:
Done